src/components/AframeScene/AframeScene.jsx

```diff
@@ -94,4 +95,4 @@ function AframeScene({ models, sliders }) {
   );
 }
 
-export default AframeScene;
+export default memo(AframeScene, (prevProps, nextProps) => isEqual(prevProps, nextProps));
```

Comment:
```suggestion
export default memo(AframeScene, isEqual);
```
I think this should work